stock_valuation_account_manual_adjustment/wizards/mass_create_view.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="view_stock_valuation_account_mass_adjust"
+                model="ir.ui.view">
+            <field name="name">Mass Adjust Stock Valuation Account Discrepancies</field>
+            <field name="model">stock.valuation.account.mass.adjust</field>
+            <field name="type">form</field>
+            <field name="arch" type="xml">
+                 <form string="Reconcile Inventory Account">
+                     <group>
+                         <field name="increase_account_id"/>
+                         <field name="decrease_account_id"/>
+                         <field name="journal_id"/>
+                         <field name="remarks"/>
+                     </group>
+                     <newline/>
+                     <group colspan="2">
+                         <button name="process"
```

Comment:
If a company starts using accounting only, when they switch to stock accounting they should open a new GL account to manage inventory valuation, of course! Then they will need to credit the balance from the old and debit into the new. Same procedure as above, but replace "opening balance equity" with "old inventory account". :)

Thanks for the feedback!